src/dune/assets.ml

```diff
@@ -0,0 +1,71 @@
+let jbuild_plugin_ml = {jbp|let () =
+  Hashtbl.add Toploop.directive_table "require"
+    (Toploop.Directive_string ignore);
+  Hashtbl.add Toploop.directive_table "use"
+    (Toploop.Directive_string
+       (fun _ ->
+         failwith "#use is not allowed inside a dune file in OCaml syntax"));
+  Hashtbl.add Toploop.directive_table "use_mod"
+    (Toploop.Directive_string
+       (fun _ ->
+         failwith "#use is not allowed inside a dune file in OCaml syntax"))
+
+module V1 = struct
+  (*$ V1.Vars $*)
+  let context = "%{context}"
```

Comment:
I suggest to write `let context = "dummy"`. The current syntax makes it look like `%{context}` is a variable, which it is not